rcl/test/rcl/test_events.cpp

```diff
@@ -202,6 +200,20 @@ class CLASSNAME (TestEventFixture, RMW_IMPLEMENTATION) : public ::testing::Test
     EXPECT_EQ(ret, RCL_RET_OK) << rcl_get_error_string().str;
   }
 
+  struct PrintToStringParamName
+  {
+    template<class ParamType>
+    std::string operator()(const ::testing::TestParamInfo<ParamType> & info) const
+    {
+      const auto & test_params = static_cast<const TestIncompatibleQosEventParams &>(info.param);
+      std::string error_msg = std::get<3>(test_params);
+      std::replace(error_msg.begin(), error_msg.end(), ' ', '_');
```

Comment:
why?

Reply:
I was using the error message as the testcase name, but GTest testcase names can't have spaces, so I was post processing the error message.

Anyway, this code has been removed and is no longer applicable.